### PR TITLE
complete highest_sum and lowest_sum

### PR DESCRIPTION
**Pull request by Captainlearyo**

### CODE REVIEW

File: lib/stat_tracker.rb [hunk 1]
```diff
@@ -24,23 +24,65 @@ def from_csv(path)
...
+    end
+
+    if hash.key?(i[:away_team_id])
+      hash[i[:away_team_id]] += i[:away_goals].to_i
+    else
+      hash[i[:away_team_id]] = i[:away_goals].to_i
+    end
+
+    end
+   p hash.values.max
```
Comment (JorjaF) [inline lib/stat_tracker.rb]: even though the test passes i am unsure why this put is here

File: lib/stat_tracker.rb [hunk 2]
```diff
@@ -24,23 +24,65 @@ def from_csv(path)
...
+    end
+
+    if hash.key?(i[:away_team_id])
+      hash[i[:away_team_id]] += i[:away_goals].to_i
+    else
+      hash[i[:away_team_id]] = i[:away_goals].to_i
+    end
+
+    end
+   p hash.values.min
```
Comment (JorjaF) [inline lib/stat_tracker.rb]: even though the test passes i am unsure why this put is here

**Comment by JorjaF** [comment]
there are a couple of puts that I am unsure of

**Review by JorjaF — CHANGES_REQUESTED** [review]
I am unsure why there are puts on here

**Review by bserge13 — APPROVED** [review]
Can't find anything I'd change. Good stuff!

**Review by JorjaF — APPROVED** [review]
looks good

**Pull request closed without merging**